test/test.js

```diff
@@ -1,7 +1,8 @@
 /*
- * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
  */
-const bedrock = require('bedrock');
+var bedrock = require('bedrock');
+require('bedrock-identity-http');
```

Comment:
The above seem like copy+paste mistakes -- reverting `var` to `const` and `2017` to `2016`.